guides/common/modules/con_managing-container-images.adoc

```diff
@@ -4,9 +4,13 @@
 With {Project}, you can import container images from various sources and distribute them to external containers by using content views.
 
 ifndef::orcharhino[]
-For information about containers for {RHEL} Atomic Host 7, see {RHDocsBaseURL}red_hat_enterprise_linux_atomic_host/7/html/getting_started_with_containers/index[Getting Started with Containers] in _{RHEL} Atomic Host 7_.
+.Additional resources
 
-For information about containers for {RHEL} 8, see {RHELDocsBaseURL}8/html-single/building_running_and_managing_containers/index[_{RHEL}{nbsp}8 Building, running, and managing containers_].
+* For information about containers for {RHEL} 10, see https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/10-beta/html-single/building_running_and_managing_containers/index[_{RHEL}{nbsp}10 Building, running, and managing containers_].
 
-For information about containers for {RHEL} 9, see {RHELDocsBaseURL}9/html-single/building_running_and_managing_containers/index[_{RHEL}{nbsp}9 Building, running, and managing containers_].
+* For information about containers for {RHEL} 9, see {RHELDocsBaseURL}9/html-single/building_running_and_managing_containers/index[_{RHEL}{nbsp}9 Building, running, and managing containers_].
+
+* For information about containers for {RHEL} 8, see {RHELDocsBaseURL}8/html-single/building_running_and_managing_containers/index[_{RHEL}{nbsp}8 Building, running, and managing containers_].
+
+* For information about containers for {RHEL} Atomic Host 7, see {RHDocsBaseURL}red_hat_enterprise_linux_atomic_host/7/html/getting_started_with_containers/index[Getting Started with Containers] in _{RHEL} Atomic Host 7_.
```

Comment:
RHEL AH is EOL since 2020: https://docs.redhat.com/en/documentation/red_hat_enterprise_linux_atomic_host/7/html/release_notes/index

Can we remove that if we're already updating the list here?

Reply:
+1